lookout/style/format/analyzer.py

```diff
@@ -93,7 +93,7 @@ def group_line_nodes(y, y_pred, vnodes_y, rule_winners):
                         find_deleted_lines(prev_file, file),
                     )))
                 fe = FeatureExtractor(language=lang, **rules.origin_config["feature_extractor"])
-                res = fe.extract_features([file], [lines])
+                res = fe.extract_features([file], [lines] if lines is not None else None)
```

Comment:
Can you please modify `extract_features` to skip `None` lines instead. That makes more sense, because some files can contain line ranges and some not.